cli/command/environment.go

```diff
@@ -223,6 +243,27 @@ func (e *EnvironmentCommand) list(c *cli.Context) error {
 	return e.printer.PrintEnvironmentSummaries(environmentSummaries...)
 }
 
+func (e *EnvironmentCommand) logs(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "ENVIRONMENT_NAME")
+	if err != nil {
+		return err
+	}
+
+	environmentID, err := e.resolveSingleEntityIDHelper("environment", args["ENVIRONMENT_NAME"])
+	if err != nil {
+		return err
+	}
+
+	query := buildLogQueryHelper(c.String("start"), c.String("end"), c.Int("tail"))
```

Comment:
Would it make sense here to potentially save a round trip by validating the `start` & `end` if they were specified?

Reply:
If we implement this, it should be done in the  `client` package in the same way we do request model validation there.

Reply:
I have decided to implement a similar validation solution in the `client` package by using the `controllers.parseLoggingQuery` helper function. I've made it exportable and call it within the `client` package validate.